subgraph.template.yaml

```diff
@@ -1239,6 +1243,46 @@ dataSources:
           handler: handleRemoveHoldersOfProjectsV1
       file: ./src/minter-suite-mapping.ts
   {{/minterHolderV1Contracts}}
+{{#minterHolderV2Contracts}}
+  - kind: ethereum/contract
+    name: 'MinterHolderV2-{{address}}'
+    network: {{network}}
+    source:
+      address: '{{address}}'
+      abi: MinterHolderV2
+      startBlock: {{startBlock}}
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.5
+      language: wasm/assemblyscript
+      entities:
+        - Project
+        - Contract
+        - MinterFilter
+        - Minter
+        - ProjectMinterConfiguration
+      abis:
+        - name: MinterHolderV2
+          file: ./abis/MinterHolderV2.json
+        - name: IFilteredMinterV0
+          file: ./abis/IFilteredMinterV0.json
+      eventHandlers:
+        - event: PricePerTokenInWeiUpdated(indexed uint256,indexed uint256)
+          handler: handlePricePerTokenInWeiUpdated
+        - event: ProjectCurrencyInfoUpdated(indexed uint256,indexed address,string)
+          handler: handleProjectCurrencyInfoUpdated
+        - event: RegisteredNFTAddress(indexed address)
+          handler: handleRegisteredNFTAddressV2
+        - event: UnregisteredNFTAddress(indexed address)
+          handler: handleUnregisteredNFTAddressV2
+        - event: AllowedHoldersOfProjects(indexed uint256,address[],uint256[])
+          handler: handleAllowHoldersOfProjectsV2
+        - event: RemovedHoldersOfProjects(indexed uint256,address[],uint256[])
+          handler: handleRemoveHoldersOfProjectsV2
+        - event: DelegationRegistryUpdated(address)
+          handler: handleHolderV2DelegationRegistryUpdated
```

Comment:
same comment as MinterMerkleV3 - can we make this come from the interface `IFilteredMinterHolderV1`?